docs/reference.rst

```diff
@@ -60,6 +60,28 @@ Example configuration:
   firmware. It will be matched against the ``compatible`` string defined in the
   update manifest.
 
+``min-bundle-version`` (optional)
+  An optional user-defined version string that follows the
+  `semantic versioning scheme <https://semver.org>`_.
+  The 'version core' and '-pre_release' parts from the ``version`` string in the
+  update manifest is compared against. This allows to roll out updates with some
+  (breaking) change, which set the ``min-bundle-version`` accordingly, and
+  through that disallow the installation of bundles with a prior version -
+  which would render the device inoperable.
```

Comment:
```suggestion
  The 'version core' and '-pre_release' parts from the ``version`` string in the
  update manifest are compared against the configured min-bundle-version.
  The update will be rejected if the bundle manifest version is smaller/older. 
  
  This allows to roll out updates with some (breaking) changes by shipping them with
  a system.conf that sets the ``min-bundle-version`` accordingly.
  Once the updated system runs, it will prevent the installation of future updates with smaller/older versions.
```